CuratedContent/WhatIsDevOps.md

```diff
@@ -0,0 +1,31 @@
+# What is DevOps?
+
+## Mueller, Ernest, 2010, https://theagileadmin.com/what-is-devops
+
+DevOps has become a catch-all term in agile organizations, but defiing what it *is* turns out not to be simple.  This blog post attempts to sort through the various ideas surrounding DevOps and bring some clarity to what is, at heart, a large and nuanced concept.
```

Comment:
Oh, I see, when you write "This blog post..." you are referring to the article you have reviewed and not to *this* artilcle. Hmm...maybe just adjust to read "Mueller's blog post..."

Reply:
Nice idea @markcmiller86 - Will do